google/datalab/stackdriver/monitoring/_utils.py

```diff
@@ -14,15 +14,18 @@
 
 from __future__ import absolute_import
 
-import google.cloud.monitoring
+# google.cloud.monitoring used to have a single Client, but
+# it has now been broken up into multiple subclients. Most
+# of the usages I found aligned with the MetricServiceClient,
+# but other uses, such as those in group_tests.py, require a
+# GroupServiceClient, and there are potentially others.
```

Comment:
I'm not sure I understand this comment. If we're not importing `GroupServiceClient`, what does the comment add?